### DISCUSSION TITLE
Periodically update composer.json

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3809] [Ideas]
In my test environment I periodically run ```composer update```. The **composer.json** file changes with new versions. When I get the commits from the OpenMage repository, this file is also retrieved, which obviously has an older version. Because there are many recordings that PHPStorm asks me to analyze, I accept the OpenMage version and then update with composer again.

Is there any way to keep both composer.json and composer.lock updated periodically in the OpenMage repository, so that whoever downloads these files gets the latest version of the packages inside?

**Comment by Flyingmana** [comment]
in the demo repository is a workflow to run a composer update, this could get reused if its wished.

**Comment by fballiano** [comment]
weird, I thought dependabot was taking care of this... I don't know why this is not happening but in the meanwhile I've created https://github.com/OpenMage/magento-lts/pull/3817
